src/sentry/issues/ignored.py

```diff
@@ -21,4 +25,72 @@ def handle_archived_until_escalating(
         remove_group_from_inbox(group, action=GroupInboxRemoveAction.IGNORED, user=acting_user)
     # TODO(snigdha): create a forecast for this group
 
-    return
+    # TODO(snigdha): we should be able to remove this in a future refactor
+    return {
+        "ignoreCount": None,
+        "ignoreUntil": None,
+        "ignoreUserCount": None,
+        "ignoreUserWindow": None,
+        "ignoreWindow": None,
```

Comment:
just curious - why is this return needed? also why are the values set to `None` instead of just returning `{}`?

Reply:
The next chunk of code in `update_groups` expects the `ignoreX` fields to be populated. We could just set the values in the next block, but this keeps that abstracted. I have a few ideas on how to make this better as we go from ongoing->archive as well.